test/unit/runner.spec.js

```diff
@@ -375,6 +380,7 @@ describe('Runner', function() {
 
     it('should not emit "end" if suite bail is not true', function(done) {
       var hook = new Hook();
+      hook.parent = suite;
       var err = {};
```

Comment:
Why is this faking `Error`s instead of making them?

Reply:
It works both ways.

Reply:
Then for clarity of intent, create an `Error` instance rather than relying on internal processing to do it for you...